packages/gin/challenge-1-basic-routing/submissions/solshuneo/solution.go

```
@@ -0,0 +1,229 @@
+	Message string      `json:"message,omitempty"`
+	Error   string      `json:"error,omitempty"`
+	Code    int         `json:"code,omitempty"`
+}
+
+// In-memory storage
+var users = []User{
+	{ID: 1, Name: "John Doe", Email: "john@example.com", Age: 30},
+	{ID: 2, Name: "Jane Smith", Email: "jane@example.com", Age: 25},
+	{ID: 3, Name: "Bob Wilson", Email: "bob@example.com", Age: 35},
+}
+var nextID = 4
```

golangci-lint (2.12.2)

[error] var nextID is unused

(unused)